sdk/storage/src/table/requests/delete_table_builder.rs

```diff
@@ -42,7 +42,7 @@ impl<'a> DeleteTableBuilder<'a> {
             &Method::DELETE,
             &|mut request| {
                 request = add_optional_header(&self.client_request_id, request);
-                request = request.header("Content-Type", "application/json");
+                request = request.header("Accept", "application/json");
```

Comment:
1. I would include the minimalodata line as well, which reduces the response payload size. We do that in Cognitive Search, at least. Not sure about other Storage SDKs (track 2) currently, though.
2. Not in this PR, but this is something we should probably just default. IIRC, most of our other track 2 SDKs do that since we mostly deal with JSON. Client implementations can override that, typically via the generator that emits most of the more recent internal implementations we wrap.

Reply:
Good idea. I'd like to keep this to just the bug fix, so I put the recommendations in issue #298.